packages/config/tests/validate/tests.js

```diff
@@ -108,6 +108,22 @@ test('build.context.CONTEXT: object', async (t) => {
   await runFixture(t, 'build_context_nested_object')
 })
 
+test('functions: object', async (t) => {
+  await runFixture(t, 'function_config_invalid_1')
+})
+
+test('functions.*: object', async (t) => {
+  await runFixture(t, 'function_config_invalid_4')
+})
+
+test('functions.*.js_external_modules: array of strings', async (t) => {
+  await runFixture(t, 'function_config_invalid_2')
+})
+
+test('functions.*.js_ignored_modules: array of strings', async (t) => {
+  await runFixture(t, 'function_config_invalid_3')
```

Comment:
[sand] Should the test fixture name be more descriptive?

Reply:
Changed in https://github.com/netlify/build/pull/2390/commits/4c6f22bee2f1f412be264060f1c0f3a73dd006b3. Let me know if that's better.

Reply:
:+1: